src/SonarQube.Bootstrapper/Program.cs

```diff
@@ -32,17 +32,18 @@ public static class Program
         public static int Main(string[] args)
         {
             var logger = new ConsoleLogger(includeTimestamp: false);
+            return Execute(args, logger);
```

Comment:
I moved the #if block inside the _Execute_ method (there was no particular reason for it to be outside, and this way it will be covered by a test).